sunpy/map/header_helper.py

```diff
@@ -49,8 +49,9 @@ def make_fitswcs_header(data, coordinate,
 
     Parameters
     ----------
-    data : `~numpy.ndarray`
-        Array data of Map for which a header is required.
+    data : `~numpy.ndarray` or `tuple`
+        Array data of Map for which a header is required, or the shape of an
```

Comment:
sorry for being pedantic - but I think it would be best to say 'or that shape of _the_ data array`